src/components/molecules/ScreenTitleView/ScreenTitleView.tsx

```diff
@@ -0,0 +1,15 @@
+import styles from "./ScreenTitleView.module.scss";
+
+interface ScreenTitleViewProps {
+  mainTitle?: string;
+}
+
+const ScreenTitleView = ({ mainTitle }: ScreenTitleViewProps) => {
```

Comment:
I wonder if we can use this one instead https://ant.design/components/page-header/ ?